cli/azd/internal/repository/app_init.go

```diff
@@ -1081,6 +1109,59 @@ func promptSpringBootVersion(console input.Console, ctx context.Context) (string
 	}
 }
 
+func promptBindingDestination(console input.Console, ctx context.Context, bindingName string) (string, error) {
+	for {
+		destination, err := console.Prompt(ctx, input.ConsoleOptions{
+			Message: fmt.Sprintf("Input the value for spring.cloud.stream.bindings.%s.destination", bindingName),
+			Help:    "Hint: Azure Eventhubs Name, please also ensure application properties is well configured.",
```

Comment:
Could you please give more information about `ensure application properties is well configured`. For example:

```
These properties are required:
1. xxx
2. xxx
```

Reply:
My intention is to remind that customer should also specify this value in the application.yml (explicitly or implicitly). If they just specify a placeholder but not specify the actual value in the application runtime, then it will failed.

Reply:
Per offline sync, the better way to deal with this kind of case is to prompt user to update application properties with a valid value, then `azd init` again. So, here we just prompt and then exit.